shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/components/SwipeBackIOSNavigationHandler.kt

```diff
@@ -0,0 +1,66 @@
+package network.bisq.mobile.presentation.ui.components
+
+import androidx.compose.foundation.gestures.detectHorizontalDragGestures
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import network.bisq.mobile.domain.getPlatform
+
+// TODO:
+// Patch work to handle horizontal swipe in iOS and pop back
+// Will remove this once compose officially has this support for iOS
+// Ref links:
+// - https://github.com/adrielcafe/voyager/issues/144
+// - https://trycatchdebug.net/news/1426361/native-ios-swipe-back-gesture-in-compose
+@Composable
+fun SwipeBackIOSNavigationHandler(
```

Comment:
Nice hack Buddha

glad to see the basic kmp Platform example got its real use finally :D 

ideally the `isIos` question should be passed to a presenter instead of the view finding out by calling a global model related getPlatform itself.